### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by szaudowsky** [eclipse/paho.mqtt.golang#466]
Hello, I'd like to ask when you're going to release new version of `paho.mqtt.golang` ?
I saw you merged a lot of things from last release which was last year.

@MattBrittan @alsm

**Comment by MattBrittan** [comment]
Duplicate of #343 - sorry cant give you a clear answer. I am waiting on commit access (with eclipse foundation currently but is taking a while) and plan on releasing a new release soon after that is granted. I believe that @master is solid; a few issues have been logged but unfortunately these have insufficient info to enable investigation (the ones I have been able to investigate recently turned out to be issues with the users code).

**Comment by szaudowsky** [comment]
Ok thanks a lot for your answer, and I hope that you'll get access soon. Sorry for duplication, I haven't seen that, my mistate :)
I'm closing this ticket.
